e2e-final/scripts/create_ins-bak.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
```

Comment:
Is the file name `create_ins-bak.sh` intended?

Reply:
Renamed